packages/react-dom/src/__tests__/DOMPropertyOperations-test.js

```diff
@@ -37,25 +39,34 @@ describe('DOMPropertyOperations', () => {
   ).set;
 
   describe('setValueForProperty', () => {
-    it('should set values as properties by default', () => {
+    it('should set values as properties by default', async () => {
       const container = document.createElement('div');
-      ReactDOM.render(<div title="Tip!" />, container);
+      const root = ReactDOMClient.createRoot(container);
+      await act(() => {
+        root.render(<div title="Tip!" />);
+      });
```

Comment:
There's some value in not adding too many abstractions for tests which is why I started with 0 new abstractions here to see how it looks.

Wondering what others think about verbosity vs. avoiding extra abstractions.

Reply:
We're generally against test utils because abstractions can be hard to change and it makes it difficult to debug or understand the use case when a test fails, which is why all the existing tests are verbose (some extremely so) even though we could have created a util. 

The verbosity is also nice because it serves as documentation for how the APIs will be used by users, without needing to unwrap the abstractions we use in our tests.